.circleci/config.yml

```diff
@@ -191,15 +214,24 @@ workflows:
 
 
 
-#      - e2eTestCurPRMultiAZ:
-#          requires:
-#          - pr
+      - e2eTestCurPRMultiAZ:
```

Comment:
Since we made a new release, we can now run multi AZ tests with both `wip` and `current`.